<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QMainWindow" name="calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>301</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_c">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_del">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>80</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>del</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_radical">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>âˆš</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_plus">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>80</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>155</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>155</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_8">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>155</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_minus">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>155</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_5">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>230</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_multiply">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>230</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>305</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>305</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>305</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_divide">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>305</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>380</y>
       <width>149</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dot">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>380</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_equals">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>380</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>300</width>
       <height>59</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
